satellite/satellitedb/projectaccounting.go

```diff
@@ -329,7 +329,7 @@ func (db *ProjectAccounting) GetBucketTotals(ctx context.Context, projectID uuid
 	countQuery := db.db.Rebind(`SELECT COUNT(DISTINCT bucket_name)
 		FROM bucket_bandwidth_rollups
 		WHERE project_id = ? AND interval_start >= ? AND interval_start <= ?
-		AND CAST(bucket_name as TEXT) LIKE ?`)
```

Comment:
bucket_name is a bytea. I think you should convert/encode it to text before comparing
try to use ```encode(data bytea, format text)```

Reply:
much better solution is to encode `search` string into bytes